api/tests/routes/backoffice/finance_test.py

```diff
@@ -60,21 +79,128 @@ class ListIncidentsTest(GetEndpointHelper):
+
+    def test_list_incident_by_collective_booking_id(self, authenticated_client, incidents):
+        searched_id = str(incidents[1].booking_finance_incidents[0].collectiveBookingId)
+
+        with testing.assert_num_queries(self.expected_num_queries):
+            response = authenticated_client.get(url_for(self.endpoint, q=searched_id))
+            assert response.status_code == 200
+
+        rows = html_parser.extract_table_rows(response.data)
+        assert len(rows) == 1
+        assert rows[0]["ID"] == str(incidents[1].id)
+        assert rows[0]["Statut de l'incident"] == "Terminé"
+        assert rows[0]["Type de résa"] == "Collective"
+        assert rows[0]["Nb. Réservation(s)"] == str(len(incidents[1].booking_finance_incidents))
+        assert rows[0]["Montant total"] == filters.format_cents(incidents[1].due_amount_by_offerer)
+        assert rows[0]["Structure"] == incidents[1].venue.managingOfferer.name
+        assert rows[0]["Porteur de l'offre (lieu)"] == incidents[1].venue.name
+        assert rows[0]["Origine de la demande"] == incidents[1].details["origin"]
+
+    def test_list_incident_by_token(self, authenticated_client, incidents):
+        searched_id = incidents[0].booking_finance_incidents[0].booking.token
+
+        with testing.assert_num_queries(self.expected_num_queries):
+            response = authenticated_client.get(url_for(self.endpoint, q=searched_id))
+            assert response.status_code == 200
+
+        rows = html_parser.extract_table_rows(response.data)
+        assert len(rows) == 1
+        assert rows[0]["ID"] == str(incidents[0].id)
+        assert rows[0]["Statut de l'incident"] == filters.format_finance_incident_status_form(incidents[0].status)
+        assert rows[0]["Type de résa"] == "Individuelle"
+        assert rows[0]["Nb. Réservation(s)"] == str(len(incidents[0].booking_finance_incidents))
+        assert rows[0]["Montant total"] == filters.format_cents(incidents[0].due_amount_by_offerer)
+        assert rows[0]["Structure"] == incidents[0].venue.managingOfferer.name
+        assert rows[0]["Porteur de l'offre (lieu)"] == incidents[0].venue.name
+        assert rows[0]["Origine de la demande"] == incidents[0].details["origin"]
+
+    def test_list_incident_by_status(self, authenticated_client, incidents):
+        status = incidents[0].status
+
+        with testing.assert_num_queries(self.expected_num_queries):
+            response = authenticated_client.get(url_for(self.endpoint, status=[status.name]))
+            assert response.status_code == 200
+
+        rows = html_parser.extract_table_rows(response.data)
+        assert len(rows) == 1
+
+    def test_list_incident_by_structures(self, authenticated_client, incidents):
+        offerer = incidents[0].venue.managingOfferer
+
+        with testing.assert_num_queries(self.expected_num_queries):
+            response = authenticated_client.get(url_for(self.endpoint, offerer=offerer.id))
```

Comment:
Attention à ne pas risquer une requête supplémentaire due au test
```suggestion
        offerer_id = incidents[0].venue.managingOffererId

        with testing.assert_num_queries(self.expected_num_queries):
            response = authenticated_client.get(url_for(self.endpoint, offerer=offerer_id))
```